tests/fixtures/file_2.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
same with files, don't replace old files - just add new ones with `nested` in name